src/transformers/models/electra/configuration_electra.py

```diff
@@ -141,6 +145,7 @@ def __init__(
         summary_last_dropout=0.1,
         pad_token_id=0,
         position_embedding_type="absolute",
+        classifier_dropout: Optional[float] = None,
```

Comment:
```suggestion
        classifier_dropout= None,
```
Let's not add type annotations just here. We can add them everywhere, but this should go in a separate PR.